inferno-framework/tutorial-test-kit
# FI-3166: Main inferno upgrade

## Summary
This PR updates the Inferno template so that the tutorial uses more up to date versions of everything. 

## Testing
Run locally with Docker:
- Run `./setup.sh` in this repository
- Run `./run.sh` in this repository
- Navigate to `http://localhost`
- Run the test kit with the following inputs:
    - FHIR Base Server URL: `https://inferno.healthit.gov/reference-server/r4`
    - Patient ID: `85`
- You should get the following results:
<img width="298" alt="image" src="https://github.com/user-attachments/assets/43840ab5-bb68-4d95-ba53-53d3358c35ca">
